### ISSUE TITLE
Render train route with bicycle allowed

### ISSUE THREAD

**Issue by Florimondable** [cyclosm/cyclosm-cartocss-style#523]
It's good to have railway, it's better if we can take our bicycle onboard.

tags are :
route=train
bicycle=yes

https://www.openstreetmap.org/relation/5572388
https://www.openstreetmap.org/relation/5569012

bicycle=limited also ?
www.openstreetmap.org/relation/5922256

**Comment by Phyks** [comment]
Indeed. Seems difficult to render since this is for routes, not ways. And multiple routes could use the same railway :/

**Comment by Florimondable** [comment]
https://wiki.openstreetmap.org/wiki/Key:passenger can be used to oder rendering

here is a try, rail highlighted in light blue :
![Screenshot_20210222_194059](https://user-images.githubusercontent.com/47089717/108754337-1840ea00-7546-11eb-89cd-cb24ffecdbc4.png)
![Screenshot_20210222_193844](https://user-images.githubusercontent.com/47089717/108754338-18d98080-7546-11eb-9034-9dc8f926fef0.png)

**Comment by quasart** [comment]
Very nice feature at low zoom. But at mid/high zoom, rail is not good for  path planing.

That’s why i think blue rails look too much like a cyclable way.
